docs/source/chapters/about.rst

```diff
@@ -84,37 +81,34 @@ written by Daniel Wendt and is now hosted on the pgRouting project site.
 
 osm2pgrouting is available under the GPLv2 license.
 
-Wiki: https://github.com/pgRouting/osm2pgrouting/wiki
+:Wiki: https://github.com/pgRouting/osm2pgrouting/wiki
 
 
 OpenStreetMap Overview
 -------------------------------------------------------------------------------
 
 .. image:: /images/osm_logo.png
     :align: center
-    :target: https://live.osgeo.org/en/overview/osm_dataset_overview.html
+    :target: http://www.openstreetmap.org
 
 
-"OpenStreetMap is a project aimed squarely at creating and providing free
-geographic data such as street maps to anyone who wants them. The project was
-started because most maps you think of as free actually have legal or technical
-restrictions on their use, holding back people from using them in creative,
-productive or unexpected ways."
+"OpenStreetMap (OSM) is dedicated to creating and providing geographic data, such as street maps, worldwide, for free. Most maps considered "free" actually have legal or technical restrictions on their use. These restrictions hold back anyone from using them in creative, productive or unexpected ways, and make every map a silo of data and effort."
 
 (Source: http://wiki.openstreetmap.org/index.php/Press)
 
-OpenStreetMap is an adequate  data source for pgRouting, because has no
+OpenStreetMap is an adequate  data source for pgRouting, because it has no
 technical restrictions in terms of processing the data. Data availability still
 varies from country to country, but the worldwide coverage is improving day by
 day.
 
 OpenStreetMap uses a topological data structure:
 
 * Nodes are points with a geographic position.
-* Ways are lists of nodes, representing a poly line or polygon.
+* Ways are lists of nodes, representing a polyline or polygon.
 * Relations are groups of nodes, ways and other relations which can be assigned
   certain properties.
 * Properties can be assigned to nodes, ways or relations and consist of
-  :code:`name=value` pairs.
+  :code:`name = value` pairs.
 
-OpenStreetMap website: http://www.openstreetmap.org
+:Website: http://www.openstreetmap.org
+:OSGeoLive: https://live.osgeo.org/en/overview/osm_dataset_overview.html
```

Comment:
Not so sure about using the overview link of OSGeoLive

Reply:
Okay. It was added because pgRouting section was having link to OSGeoLive Overview, so I added it in OpenStreetMap too. 
Please suggest what to do.

Reply:
I am not sure, maybe @dkastl can comment on this one

Reply:
Okay. Sure. For now, I will keep it as earliar and remove the OSGeoLive link and push the changes. After getting @dkastl suggestions, I will edit it again, if required.

Reply:
I am fine with removing it.